spec/lib/vmdb/loggers_spec.rb

```diff
@@ -24,6 +25,15 @@ def in_container_env(example)
     example.run
   ensure
     ENV['CONTAINER'] = old_env
+    undef_env(:@is_appliance, :@is_podified)
```

Comment:
I thought I had to do this as part of https://github.com/ManageIQ/manageiq/pull/21724 also but @Fryguy pointed me to `EvmSpecHelper.clear_caches` that already clears `MiqEnvironment::Command` can you tell if that isn't working for some reason in this instance?

Reply:
thanks. I couldn't think of the name of `clear_caches` at the time and knew better than to do this.
This time I got in the real fix